src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -499,9 +499,20 @@ namespace DataHeatBalance {
         Array1D<Real64> ReflBack;   // Back reflectance at normal incidence
     };
 
-    struct SpaceData
+    struct ZoneSpaceData
+    {
+        // Base class for zones and spaces.
+        // For now, only including fields that are new to Space to avoid excess changes
+        // due to case differences between existing space and zone field names.
+        std::string Name;
+        Real64 CeilingHeight = DataGlobalConstants::AutoCalculate; // Ceiling Height entered by user [m] or calculated
+        Real64 Volume = DataGlobalConstants::AutoCalculate;        // Volume entered by user [m3] or calculated
+        Real64 ExtGrossWallArea = 0.0;                             // Exterior Wall Area for Zone (Gross)
+        Real64 ExteriorTotalSurfArea = 0.0;                        // Total surface area of all exterior surfaces for Zone
+    };
+
+    struct SpaceData : ZoneSpaceData
```

Comment:
SpaceData and ZoneData each have 3 floor areas, UserEnteredFloorArea, FloorArea, and CalcFloorArea. Can you imagine having UserEnteredCoilCapacity, CoilCapacity and CalcCoilCapacity. Could probably get rid of at least one of these in the future.